### PR TITLE
Fix broken rosbags

### PR DESCRIPTION
**Pull request by BenStarmerSmith**
## Proposed changes

Added a new function to gather all active rosbags and fix them.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [x] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [x] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

**Review by maxzieba — CHANGES_REQUESTED** [review]
With your added method, you can remove the parts in the previous version regarding removing active rosbags as you don't expect to have them anymore. Additionaly have a look maybe at the rosbag API (example: https://answers.ros.org/question/298525/reindex-bag-using-rosbag-api/)

File: sr_logging_common/logging/bag_rotate.py [hunk 1]
```diff
@@ -15,34 +15,53 @@
...
+        string = "Fixed the rosbag files:"
+        for filename in active_rosbags:
+            string += f"  {filename}"
+        rospy.loginfo(string)
+
+
 if __name__ == '__main__':
     rospy.init_node('bag_rotate', anonymous=True)
     desired_bag_number = rospy.get_param('~bag_files_num')
     path = rospy.get_param('~bag_files_path')
```
Comment (maxzieba) [inline sr_logging_common/logging/bag_rotate.py]: Add a try/except or add default values.

File: sr_logging_common/logging/bag_rotate.py [hunk 2]
```diff
@@ -15,34 +15,53 @@
...
-import os
+from os import listdir, remove
+from os.path import getctime, join, exists
 import time
 import rospy
+import subprocess
 
 
 def remover(desired_bag_number, path):
     CONST_MAX_ACTIVE_BAGS_ALLOWED = desired_bag_number//2
```
Comment (bdenoun) [inline sr_logging_common/logging/bag_rotate.py]: You can remove this if you are not using it anymore

File: sr_logging_common/logging/bag_rotate.py (hunk 1 above)
Comment (bdenoun) [reply sr_logging_common/logging/bag_rotate.py]: You could also just check that the bag file path does exist before calling any of your functions

File: sr_logging_common/logging/bag_rotate.py (hunk 2 above)
Comment (BenStarmerSmith) [reply sr_logging_common/logging/bag_rotate.py]: I am using it now, there was no way of fixing rosbags using the python package

Comment (bdenoun) [reply sr_logging_common/logging/bag_rotate.py]: So sorry, I misclicked, I meant from __future__ thingy

**Review by bdenoun — APPROVED** [review]

**Review by maxzieba — APPROVED** [review]

**Pull request merged**